docker/aws-eks-utils/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM alpine:3.10.1
```

Comment:
3.13 is the latest alpine version

Reply:
I changed the alpine version to 3.13.5 for `aws-eks-utils` and `terraform-utils` dockerfiles.